web/packages/api/src/xcmBuilder.ts

```diff
@@ -23,6 +31,34 @@ export function parachainLocation(paraId: number) {
     }
 }
 
+export function accountId32Location(hexAddress: string) {
+    return {
+        parents: 0,
+        interior: { x1: [{ accountId32: { id: hexAddress } }] },
+    }
+}
+
+export function kusamaAssetHubLocation(parachainId: number) {
+    return {
+        parents: 2,
+        interior: { x2: [{ GlobalConsensus: { Kusama: null } }, { parachain: parachainId }] },
+    }
+}
+
+export function polkadotAssetHubLocation(parachainId: number) {
+    return {
+        parents: 2,
+        interior: { x2: [{ GlobalConsensus: { Polkadot: null } }, { parachain: parachainId }] },
+    }
+}
+
+export function dotLocationOnKusamaAssetHubLocation() {
```

Comment:
Just use constants instead of lambda.

Reply:
Addressed in https://github.com/Snowfork/snowbridge/pull/1463/commits/f309c26ea81cff0207f542a9ba8cfecc82e561de.